friends/src/App.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import axios from "axios";
+
+import FriendsList from "./components/FriendsContainer/FriendsList";
+import PostFriendForm from "./components/FriendsContainer/PostFriendForm";
+
+import "./App.css";
+
+class App extends React.Component {
+  state = {
+    friends: []
+    // newFriend: {
+    //   id: null,
+    //   name: "",
+    //   age: null,
+    //   email: ""
+    // }
+  };
+
+  componentDidMount() {
+    axios
+      .get("http://localhost:5000/friends")
+      .then(res => {
+        this.setState({ friends: res.data });
+      })
+      .catch(err => console.log(err));
+    console.log(this.state.friends);
+  }
+
+  postNewFriend = friend => {
+    axios
+      .post("http://localhost:5000/friends", friend)
+      .then(res => console.log(res))
```

Comment:
If you update state in this method, there will be a re-render because of the React Lifecycle and it will update without a page refresh. `.then(res => this.setState({friends: res.data}))`